src/js/languages/default.json

```diff
@@ -72,17 +72,21 @@
     "label": "View"
   },
   "fileInput": {
+    "alert": {
+      "maxSize": "Attention, this file {fileName} is too large. Attach files which are no larger than {maxSize} to continue.",
+      "maxFile": "Attention, attach a maximum of {maxFile} files only. File {fileName} is beyond this limit."
```

Comment:
Suggestion for brevity:

"{fileName} exceeds limit of {maxFile} file limit."

Reply:
What's the different between this "maxFile" message and when the other "maxFile" message will be read out? Any opportunity to just reuse the existing message?